ethereumj-core/src/main/java/org/ethereum/net/eth/handler/Eth63.java

```diff
@@ -95,8 +96,9 @@ protected synchronized void processGetNodeData(GetNodeDataMessage msg) {
         );
 
         List<Value> states = new ArrayList<>();
+        RepositoryRoot repositoryRoot = ((RepositoryRoot) repository);
         for (byte[] stateRoot : msg.getStateRoots()) {
```

Comment:
Could you please also rename `stateRoot(s)` to something like `nodeKey` or `nodeHash` to avoid confusion